.tekton/build_and_trigger.yaml

```diff
@@ -85,56 +112,54 @@ spec:
         computeResources:
           requests:
             cpu: 1
+        runAfter:
+          - calculate-tag
+        taskRef:
+          name: buildah
+          kind: ClusterTask
         params:
           - name: IMAGE
             value: "quay.io/redhat-appstudio/clair-in-ci:$(tasks.calculate-tag.results.image_tag)"
           - name: BUILDER_IMAGE
             value: $(params.builder-image)
           - name: DOCKERFILE
             value: $(workspaces.source.path)/$(params.dockerfile)
-        runAfter:
-          - calculate-tag
-        taskRef:
-          name: buildah
-          kind: ClusterTask
-      - name: trigger-tests-and-push
-        computeResources:
-          requests:
-            cpu: 100mi
-        params:
-          - name: revision
-            value: "{{ revision }}"
-          - name: event_type
-            value: "{{ event_type }}"
-          - name: image_tag
-            value: $(tasks.calculate-tag.results.image_tag)
+      - name: test-clair
         runAfter:
           - build-image
         taskSpec:
-          params:
-            - name: revision
-              type: string
-            - name: event_type
-              type: string
-            - name: image_tag
-              type: string
           steps:
-            - name: tigger-workflow
-              image: registry.access.redhat.com/ubi9/python-39
-              env:
-                - name: GITHUB_TOKEN
-                  valueFrom:
-                    secretKeyRef:
-                      name: clair-in-ci-db-github-token
-                      key: github-token
+            - name: test-clair-version
+              image: quay.io/redhat-appstudio/clair-in-ci:$(tasks.calculate-tag.results.image_tag)
               script: |
                 #!/usr/bin/env bash
-                event="pull_request"
-                if [ "$(params.event_type)" == "push" ]; then
-                  event="push"
+                echo "See current version of Clair below:"
+                clair-action --version
+            - name: get-clair-output
+              image: quay.io/redhat-appstudio/clair-in-ci:$(tasks.calculate-tag.results.image_tag)
+              script: |
+                #!/usr/bin/env bash
+                echo "Test real life usage of Clair"
+                clair-action report --image-ref=registry.access.redhat.com/ubi9-minimal --db-path=/tmp/matcher.db --format=quay > /tekton/home/clairdata.json
```

Comment:
suggestion: it might be a good idea to use `tee` instead of a redirect to file here.
That way we'd be able to see the output of the command in the logs which should help with debugging issues.

Reply:
thanks